2018-03-09 10:24:42,766 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:24:42 +0800] "GET /manage/login HTTP/1.0" 200 7904 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:24:51,719 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:24:51 +0800] "GET /manage/login HTTP/1.0" 200 7904 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:25:01,993 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:01 +0800] "GET /manage/login HTTP/1.0" 200 7904 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,046 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/css/skins/dneglu.css HTTP/1.0" 200 2912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,047 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/css/bootstrap.min.css HTTP/1.0" 200 114007 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,061 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,065 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,066 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,069 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,070 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,083 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/css/login.css HTTP/1.0" 200 3912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,087 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,287 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:02,512 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:02 +0800] "GET /static/css/particles.css HTTP/1.0" 200 549 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:03,180 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:03 +0800] "GET /static/image/head_img/tubiao.png HTTP/1.0" 200 7584 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:05,133 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:05 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:25:06,129 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:25:06 +0800] "GET /favicon.ico HTTP/1.0" 200 1150 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:11,037 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:26:11 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 1048352 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:15,040 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:26:15 +0800] "- - HTTP/1.0" 0 0 "-" "-"
2018-03-09 10:26:15,040 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:26:15 +0800] "- - HTTP/1.0" 0 0 "-" "-"
2018-03-09 10:26:46,315 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /manage/login HTTP/1.0" 200 7904 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,556 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,559 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/css/skins/dneglu.css HTTP/1.0" 200 2912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,562 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,564 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,564 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,565 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/css/bootstrap.min.css HTTP/1.0" 200 114007 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,586 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/css/login.css HTTP/1.0" 200 3912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,787 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/css/particles.css HTTP/1.0" 200 549 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,790 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,814 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:46,818 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:46 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:47,878 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:47 +0800] "GET /static/image/head_img/tubiao.png HTTP/1.0" 200 7584 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:26:49,821 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:26:49 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:00,232 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:00 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:05,533 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:05 +0800] "GET /manage/login HTTP/1.0" 200 7904 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,097 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/css/skins/dneglu.css HTTP/1.0" 200 2912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,100 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,103 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,105 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,106 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/css/bootstrap.min.css HTTP/1.0" 200 114007 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,106 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,141 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/css/particles.css HTTP/1.0" 200 549 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,143 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/css/login.css HTTP/1.0" 200 3912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,157 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,379 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:06,927 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:06 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:07,522 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:07 +0800] "GET /static/image/head_img/tubiao.png HTTP/1.0" 200 7584 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:08,895 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:08 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:11,058 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:11 +0800] "GET /favicon.ico HTTP/1.0" 200 1150 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:22,750 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:27:22 +0800] "GET /api/check_version HTTP/1.1" 200 107 "-" "okhttp/3.9.1"
2018-03-09 10:27:29,350 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /manage/login HTTP/1.0" 200 7904 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:29,514 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /static/css/bootstrap.min.css HTTP/1.0" 200 114007 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:29,528 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /static/css/skins/dneglu.css HTTP/1.0" 200 2912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:29,531 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:29,533 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:29,536 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:29,541 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:29 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:30,803 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:30 +0800] "GET /static/css/particles.css HTTP/1.0" 200 549 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:30,837 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:30 +0800] "GET /static/css/login.css HTTP/1.0" 200 3912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:30,933 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:30 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:30,976 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:30 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:30,981 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:30 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:32,641 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:32 +0800] "GET /static/image/head_img/tubiao.png HTTP/1.0" 200 7584 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:27:33,117 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:27:33 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,604 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,657 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,659 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/css/skins/dneglu.css HTTP/1.0" 200 2912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,661 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,666 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,666 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,667 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/css/bootstrap.min.css HTTP/1.0" 200 114007 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,693 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/css/particles.css HTTP/1.0" 200 549 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,695 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/css/login.css HTTP/1.0" 200 3912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,697 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:24,967 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:24 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:25,172 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:25 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:27,795 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:27 +0800] "GET /static/image/head_img/tubiao.png HTTP/1.0" 200 7584 "http://5k.safego.org/manage/login" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:28,404 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:28 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.140 Safari/537.36"
2018-03-09 10:28:41,934 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:28:41 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:28:45,726 mlogging.py[line:130] INFO "61.129.6.148 - - [09/Mar/2018:10:28:45 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 133216 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:28:45,768 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:28:45 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 99912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:28:55,052 mlogging.py[line:130] INFO "61.129.6.148 - - [09/Mar/2018:10:28:55 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 70952 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:28:55,086 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:28:55 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 99912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:29:11,355 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,486 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/css/bootstrap.min.css HTTP/1.0" 200 114007 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,498 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/css/particles.css HTTP/1.0" 200 549 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,501 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/css/login.css HTTP/1.0" 200 3912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,505 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,507 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/css/skins/dneglu.css HTTP/1.0" 200 2912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,507 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,533 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,596 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,599 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,666 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,730 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:11,919 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:11 +0800] "GET /static/image/head_img/tubiao.png HTTP/1.0" 200 7584 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:14,365 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:14 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:15,885 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:15 +0800] "GET /favicon.ico HTTP/1.0" 200 1150 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:15,928 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:29:15 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 99912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:16,821 mlogging.py[line:130] INFO "119.254.236.252 - - [09/Mar/2018:10:29:16 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 206 256 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0.1; vivo Y53 Build/MMB29M)"
2018-03-09 10:29:17,024 mlogging.py[line:130] INFO "119.254.236.252 - - [09/Mar/2018:10:29:17 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 206 85141 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0.1; vivo Y53 Build/MMB29M)"
2018-03-09 10:29:17,301 mlogging.py[line:130] INFO "119.254.236.252 - - [09/Mar/2018:10:29:17 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 206 3077 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0.1; vivo Y53 Build/MMB29M)"
2018-03-09 10:29:19,940 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:29:19 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 1049800 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36"
2018-03-09 10:29:23,945 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:29:23 +0800] "- - HTTP/1.0" 0 0 "-" "-"
2018-03-09 10:29:38,756 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:38 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:29:40,440 mlogging.py[line:130] INFO "61.129.6.148 - - [09/Mar/2018:10:29:40 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 133216 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:29:40,480 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:29:40 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 99912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:29:45,861 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:45 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36 VivoBrowser/5.3.3"
2018-03-09 10:29:46,081 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:29:46 +0800] "GET /static/image/head_img/37f671a0d8071b66ecfb9b7d2a17286d.png HTTP/1.0" 200 373919 "http://5k.safego.org/static/css/login.css" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36 VivoBrowser/5.3.3"
2018-03-09 10:29:49,246 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:29:49 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 99912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36 VivoBrowser/5.3.3"
2018-03-09 10:29:50,649 mlogging.py[line:130] INFO "119.254.236.252 - - [09/Mar/2018:10:29:50 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 206 256 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0.1; vivo Y53 Build/MMB29M)"
2018-03-09 10:29:50,911 mlogging.py[line:130] INFO "119.254.236.252 - - [09/Mar/2018:10:29:50 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 206 85141 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0.1; vivo Y53 Build/MMB29M)"
2018-03-09 10:29:51,280 mlogging.py[line:130] INFO "119.254.236.252 - - [09/Mar/2018:10:29:51 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 206 3077 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0.1; vivo Y53 Build/MMB29M)"
2018-03-09 10:30:11,182 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:11 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; U; Android 4.4.4; zh-CN; HUAWEI ALE-CL00 Build/HuaweiALE-CL00) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/40.0.2214.89 UCBrowser/11.3.0.907 Mobile Safari/537.36"
2018-03-09 10:30:11,662 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:11 +0800] "GET /static/js/jquery-1.11.3.min.js HTTP/1.0" 200 95957 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 5.1.1; SM801 Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko)     		Chrome/46.0.2490.11 Mobile Safari/537.36"
2018-03-09 10:30:12,239 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:12 +0800] "GET /static/js/bootstrap.min.js HTTP/1.0" 200 34653 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; U; Android 4.4.4; zh-CN; HUAWEI ALE-CL00 Build/HuaweiALE-CL00) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/40.0.2214.89 UCBrowser/11.3.0.907 Mobile Safari/537.36"
2018-03-09 10:30:12,454 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:12 +0800] "GET /static/js/denglu.js HTTP/1.0" 200 4269 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; U; Android 6.0.1; zh-cn; OPPO R9s Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.0.0 Mobile Safari/537.36 OppoBrowser/4.2.8"
2018-03-09 10:30:12,528 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:12 +0800] "GET /static/js/common.js HTTP/1.0" 200 10085 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; U; Android 4.4.4; zh-CN; Coolpad Y82-520 Build/KTU84P) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 UCBrowser/10.9.8.738 U3/0.8.0 Mobile Safari/534.30"
2018-03-09 10:30:12,599 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:12 +0800] "GET /static/js/jquery.base64.js HTTP/1.0" 200 4498 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; U; Android 4.4.4; zh-CN; HUAWEI ALE-CL00 Build/HuaweiALE-CL00) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/40.0.2214.89 UCBrowser/11.3.0.907 Mobile Safari/537.36"
2018-03-09 10:30:12,740 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:12 +0800] "GET /static/js/jquery.md5.js HTTP/1.0" 200 10674 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; U; Android 4.4.4; zh-CN; HUAWEI ALE-CL00 Build/HuaweiALE-CL00) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/40.0.2214.89 UCBrowser/11.3.0.907 Mobile Safari/537.36"
2018-03-09 10:30:12,810 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:30:12 +0800] "GET /static/js/login.js HTTP/1.0" 200 4115 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 5.1.1; SM801 Build/LMY47V) AppleWebKit/537.36 (KHTML, like Gecko)     		Chrome/46.0.2490.11 Mobile Safari/537.36"
2018-03-09 10:30:24,046 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:30:24 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 1194600 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36 VivoBrowser/5.3.3"
2018-03-09 10:31:21,035 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:31:21 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:31:24,630 mlogging.py[line:130] INFO "61.129.6.148 - - [09/Mar/2018:10:31:24 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 70952 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:31:24,687 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:31:24 +0800] "GET /static/apk/safego_v1.2.6_20180210_release.apk HTTP/1.1" 500 99912 "http://5k.safego.org/manage/login" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:37:23,458 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:10:37:23 +0800] "GET /api/check_version HTTP/1.1" 200 107 "-" "okhttp/3.9.1"
2018-03-09 10:40:05,605 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:40:05 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 10:40:15,689 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:40:15 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36 VivoBrowser/5.3.3"
2018-03-09 10:45:03,818 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:10:45:03 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 5.1.1; vivo X7 Build/LMY47V; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 11:18:14,559 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:11:18:14 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 11:20:09,340 user_view.py[line:138] INFO function=app_user_login,args={'result': {'token': 'b086a139e2ee45a587924104dae673a1', 'data': {'phones': ['59ae7888de713e343baaf5d1', '59e7500aad01be1548e7017e'], '_id': '59e56d44ad01be04337b8a34', 'head_img_url': 'static/image/head_img/default_01.png', 'description': '物联网卡用户', 'real_name': '钟伟', 'user_password': '42ff023c3bc5982cf54e4a6ddb889f3c', 'country': '中国', 'user_name': '1064823647998', 'app_version': '1.2.6.0207 Debug', 'create_date': '2017-11-13 20:49:00', 'last_update': '2018-02-10 16:45:17', 'phone_num': '1064823647998', 'age': 0, 'nick_name': '小淘气'}}, 'random_code': 0, 'phone_num': '1064823647998', 'login_type': 'TY_PHONE', 'message': {'message': 'success', 'token': 'b086a139e2ee45a587924104dae673a1', 'data': {'phones': ['59ae7888de713e343baaf5d1', '59e7500aad01be1548e7017e'], '_id': '59e56d44ad01be04337b8a34', 'head_img_url': 'static/image/head_img/default_01.png', 'description': '物联网卡用户', 'real_name': '钟伟', 'user_password': '42ff023c3bc5982cf54e4a6ddb889f3c', 'country': '中国', 'user_name': '1064823647998', 'app_version': '1.2.6.0207 Debug', 'create_date': '2017-11-13 20:49:00', 'last_update': '2018-02-10 16:45:17', 'phone_num': '1064823647998', 'age': 0, 'nick_name': '小淘气'}}, 'user_password': '', 'user_name': '1064823647998'}
2018-03-09 11:20:09,341 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:20:09 +0800] "POST /api/login HTTP/1.1" 200 616 "-" "okhttp/3.9.1"
2018-03-09 11:20:10,303 company_module.py[line:160] INFO 用户59e56d44ad01be04337b8a34没有company_id信息
2018-03-09 11:20:10,363 base.py[line:83] INFO GET http://api.safego.org:9200/sf/daily_report/_search?size=100 [status:200 request:0.059s]
2018-03-09 11:20:10,366 security_module.py[line:1224] INFO elasticsearch query success! at 2018-03-09 11:20:10, user_id:59e56d44ad01be04337b8a34,date:None共计查询到19条记录
2018-03-09 11:20:10,367 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:20:10 +0800] "POST /api/get_daily_info HTTP/1.1" 200 124 "-" "okhttp/3.9.1"
2018-03-09 11:20:14,929 user_view.py[line:198] INFO function=process_user_info,args={'message': {'message': 'success', 'data': {'phones': ['59ae7888de713e343baaf5d1', '59e7500aad01be1548e7017e'], '_id': '59e56d44ad01be04337b8a34', 'head_img_url': 'http://safego.org:5000/static/image/head_img/default_01.png', 'description': '物联网卡用户', 'real_name': '钟伟', 'user_password': '42ff023c3bc5982cf54e4a6ddb889f3c', 'country': '中国', 'user_name': '1064823647998', 'app_version': '1.2.6.0207 Debug', 'create_date': '2017-11-13 20:49:00', 'last_update': '2018-02-10 16:45:17', 'phone_num': '1064823647998', 'age': 0, 'nick_name': '小淘气'}}, 'host_url': 'http://safego.org:5000/', 'key': 'get', 'user_id': ObjectId('59e56d44ad01be04337b8a34')}
2018-03-09 11:20:14,931 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:20:14 +0800] "POST /api/get_user_info HTTP/1.1" 200 594 "-" "okhttp/3.9.1"
2018-03-09 11:20:15,375 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:20:15 +0800] "GET /static/image/head_img/default_01.png HTTP/1.1" 200 14402 "-" "Dalvik/2.1.0 (Linux; U; Android P Build/PPP1.180208.011)"
2018-03-09 11:20:17,426 user_view.py[line:198] INFO function=process_user_info,args={'message': {'message': 'success', 'data': {'phones': ['59ae7888de713e343baaf5d1', '59e7500aad01be1548e7017e'], '_id': '59e56d44ad01be04337b8a34', 'head_img_url': 'http://safego.org:5000/static/image/head_img/default_01.png', 'description': '物联网卡用户', 'real_name': '钟伟', 'user_password': '42ff023c3bc5982cf54e4a6ddb889f3c', 'country': '中国', 'user_name': '1064823647998', 'app_version': '1.2.6.0207 Debug', 'create_date': '2017-11-13 20:49:00', 'last_update': '2018-02-10 16:45:17', 'phone_num': '1064823647998', 'age': 0, 'nick_name': '小淘气'}}, 'host_url': 'http://safego.org:5000/', 'key': 'get', 'user_id': ObjectId('59e56d44ad01be04337b8a34')}
2018-03-09 11:20:17,427 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:20:17 +0800] "POST /api/get_user_info HTTP/1.1" 200 594 "-" "okhttp/3.9.1"
2018-03-09 11:20:17,870 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:20:17 +0800] "GET /static/image/head_img/default_01.png HTTP/1.1" 200 14402 "-" "Dalvik/2.1.0 (Linux; U; Android P Build/PPP1.180208.011)"
2018-03-09 11:27:20,998 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:11:27:20 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 11:57:07,771 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:11:57:07 +0800] "GET /api/check_version HTTP/1.1" 200 107 "-" "okhttp/3.9.1"
2018-03-09 14:01:53,985 mlogging.py[line:130] INFO "223.104.255.84 - - [09/Mar/2018:14:01:53 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:40:42,537 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:40:42 +0800] "GET /api/check_version HTTP/1.1" 200 107 "-" "okhttp/3.9.1"
2018-03-09 14:41:01,625 user_view.py[line:167] INFO function=api_send_sms,args={'message': {'message': 'success'}, 'phone_num': '18706202810'}
2018-03-09 14:41:01,626 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:01 +0800] "POST /api/get_sms HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:41:23,487 user_view.py[line:138] INFO function=app_user_login,args={'result': <api.data.item_module.User object at 0x7faa11374048>, 'phone_num': '18706202810', 'random_code': 7620, 'validate_result': {'message': 'success'}, 'login_type': 'TY_PHONE', 'message': {'message': 'success', 'token': '96ed243f1410413894d3cd04c77ebc14', 'data': {'license_id': '340888888', '_id': '598d6ac2de713e32dfc74796', 'nick_name': '小笨蛋2', 'head_img_url': 'static/image/avatar/c181666905954b57aedff75ddc8d57b6.jpg', 'birth_date': '1976-12-11', 'user_password': '67531f05888af75a526b07df6c4aec32', 'country': '中国', 'user_name': '18706202810', 'valid_date': '2020-12-30 00:00:00', 'invite_code': '000', 'app_version': '1.2.8.0307 Debug', 'license_class': 'C1', 'create_date': '2017-08-11 16:28:50', 'last_update': '2018-03-08 15:49:25', 'phone_num': '18706202810', 'age': 0, 'first_issued_date': '2014-04-25 00:00:00', 'phones': ['59bb2e1aad01be2ffb0f4f0c', '59c0daf3ad01be6e562471d7', '59ae7888de713e343baaf5d1', '59b20e0dde713e1456cfdb52', '59b0dc3dde713e17164296f3'], 'license_image_url': 'static/image/license_image/f60976b00b234d71898f1d3e13165a8e.jpg', 'gender': '男', 'address': '上海市嘉定区安亭镇昌吉东路156号'}}, 'token': '96ed243f1410413894d3cd04c77ebc14', 'user_password': '', 'user_name': '18706202810'}
2018-03-09 14:41:23,489 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:23 +0800] "POST /api/login HTTP/1.1" 200 1024 "-" "okhttp/3.9.1"
2018-03-09 14:41:23,935 company_module.py[line:160] INFO 用户598d6ac2de713e32dfc74796没有company_id信息
2018-03-09 14:41:23,965 base.py[line:83] INFO GET http://api.safego.org:9200/sf/daily_report/_search?size=100 [status:200 request:0.028s]
2018-03-09 14:41:23,967 security_module.py[line:1224] INFO elasticsearch query success! at 2018-03-09 14:41:23, user_id:598d6ac2de713e32dfc74796,date:None共计查询到6条记录
2018-03-09 14:41:23,968 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:23 +0800] "POST /api/get_daily_info HTTP/1.1" 200 124 "-" "okhttp/3.9.1"
2018-03-09 14:41:31,625 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:31 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:41:31,738 data_view.py[line:244] INFO 2018-03-09 14:41:31.738302: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_14_41_29.zip' ('multipart/form-data')>)])
2018-03-09 14:41:31,849 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:31 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:41:32,223 user_view.py[line:198] INFO function=process_user_info,args={'message': {'message': 'success', 'data': {'license_id': '340888888', '_id': '598d6ac2de713e32dfc74796', 'nick_name': '小笨蛋2', 'head_img_url': 'http://safego.org:5000/static/image/avatar/c181666905954b57aedff75ddc8d57b6.jpg', 'age': 0, 'user_password': '67531f05888af75a526b07df6c4aec32', 'country': '中国', 'user_name': '18706202810', 'valid_date': '2020-12-30 00:00:00', 'invite_code': '000', 'app_version': '1.2.8.0309 Debug', 'license_class': 'C1', 'create_date': '2017-08-11 16:28:50', 'last_update': '2018-03-09 14:41:31', 'phone_num': '18706202810', 'birth_date': '1976-12-11', 'first_issued_date': '2014-04-25 00:00:00', 'phones': ['59bb2e1aad01be2ffb0f4f0c', '59c0daf3ad01be6e562471d7', '59ae7888de713e343baaf5d1', '59b20e0dde713e1456cfdb52', '59b0dc3dde713e17164296f3'], 'license_image_url': 'static/image/license_image/f60976b00b234d71898f1d3e13165a8e.jpg', 'gender': '男', 'address': '上海市嘉定区安亭镇昌吉东路156号'}}, 'host_url': 'http://safego.org:5000/', 'key': 'get', 'user_id': ObjectId('598d6ac2de713e32dfc74796')}
2018-03-09 14:41:32,225 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:32 +0800] "POST /api/get_user_info HTTP/1.1" 200 1002 "-" "okhttp/3.9.1"
2018-03-09 14:41:32,511 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:32 +0800] "GET /static/image/avatar/c181666905954b57aedff75ddc8d57b6.jpg HTTP/1.1" 200 78361 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0; CAM-AL00 Build/HONORCAM-AL00)"
2018-03-09 14:41:35,410 company_module.py[line:160] INFO 用户598d6ac2de713e32dfc74796没有company_id信息
2018-03-09 14:41:35,446 base.py[line:83] INFO GET http://api.safego.org:9200/sf/daily_report/_search?size=100 [status:200 request:0.035s]
2018-03-09 14:41:35,447 security_module.py[line:1224] INFO elasticsearch query success! at 2018-03-09 14:41:35, user_id:598d6ac2de713e32dfc74796,date:None共计查询到6条记录
2018-03-09 14:41:35,448 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:35 +0800] "POST /api/get_daily_info HTTP/1.1" 200 124 "-" "okhttp/3.9.1"
2018-03-09 14:41:44,313 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:44 +0800] "GET /api/check_version HTTP/1.1" 200 107 "-" "okhttp/3.9.1"
2018-03-09 14:41:48,419 company_module.py[line:160] INFO 用户598d6ac2de713e32dfc74796没有company_id信息
2018-03-09 14:41:48,449 base.py[line:83] INFO GET http://api.safego.org:9200/sf/daily_report/_search?size=100 [status:200 request:0.029s]
2018-03-09 14:41:48,450 security_module.py[line:1224] INFO elasticsearch query success! at 2018-03-09 14:41:48, user_id:598d6ac2de713e32dfc74796,date:None共计查询到6条记录
2018-03-09 14:41:48,451 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:48 +0800] "POST /api/get_daily_info HTTP/1.1" 200 124 "-" "okhttp/3.9.1"
2018-03-09 14:41:49,616 user_view.py[line:198] INFO function=process_user_info,args={'message': {'message': 'success', 'data': {'license_id': '340888888', '_id': '598d6ac2de713e32dfc74796', 'nick_name': '小笨蛋2', 'head_img_url': 'http://safego.org:5000/static/image/avatar/c181666905954b57aedff75ddc8d57b6.jpg', 'age': 0, 'user_password': '67531f05888af75a526b07df6c4aec32', 'country': '中国', 'user_name': '18706202810', 'valid_date': '2020-12-30 00:00:00', 'invite_code': '000', 'app_version': '1.2.8.0309 Debug', 'license_class': 'C1', 'create_date': '2017-08-11 16:28:50', 'last_update': '2018-03-09 14:41:31', 'phone_num': '18706202810', 'birth_date': '1976-12-11', 'first_issued_date': '2014-04-25 00:00:00', 'phones': ['59bb2e1aad01be2ffb0f4f0c', '59c0daf3ad01be6e562471d7', '59ae7888de713e343baaf5d1', '59b20e0dde713e1456cfdb52', '59b0dc3dde713e17164296f3'], 'license_image_url': 'static/image/license_image/f60976b00b234d71898f1d3e13165a8e.jpg', 'gender': '男', 'address': '上海市嘉定区安亭镇昌吉东路156号'}}, 'host_url': 'http://safego.org:5000/', 'key': 'get', 'user_id': ObjectId('598d6ac2de713e32dfc74796')}
2018-03-09 14:41:49,617 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:49 +0800] "POST /api/get_user_info HTTP/1.1" 200 1002 "-" "okhttp/3.9.1"
2018-03-09 14:41:50,696 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:41:52,192 user_view.py[line:198] INFO function=process_user_info,args={'message': {'message': 'success', 'data': {'license_id': '340888888', '_id': '598d6ac2de713e32dfc74796', 'nick_name': '小笨蛋2', 'head_img_url': 'http://safego.org:5000/static/image/avatar/c181666905954b57aedff75ddc8d57b6.jpg', 'age': 0, 'user_password': '67531f05888af75a526b07df6c4aec32', 'country': '中国', 'user_name': '18706202810', 'valid_date': '2020-12-30 00:00:00', 'invite_code': '000', 'app_version': '1.2.8.0309 Debug', 'license_class': 'C1', 'create_date': '2017-08-11 16:28:50', 'last_update': '2018-03-09 14:41:31', 'phone_num': '18706202810', 'birth_date': '1976-12-11', 'first_issued_date': '2014-04-25 00:00:00', 'phones': ['59bb2e1aad01be2ffb0f4f0c', '59c0daf3ad01be6e562471d7', '59ae7888de713e343baaf5d1', '59b20e0dde713e1456cfdb52', '59b0dc3dde713e17164296f3'], 'license_image_url': 'static/image/license_image/f60976b00b234d71898f1d3e13165a8e.jpg', 'gender': '男', 'address': '上海市嘉定区安亭镇昌吉东路156号'}}, 'host_url': 'http://safego.org:5000/', 'key': 'get', 'user_id': ObjectId('598d6ac2de713e32dfc74796')}
2018-03-09 14:41:52,194 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:52 +0800] "POST /api/get_user_info HTTP/1.1" 200 1002 "-" "okhttp/3.9.1"
2018-03-09 14:41:52,195 data_view.py[line:244] INFO 2018-03-09 14:41:52.195695: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_14_41_50.zip' ('multipart/form-data')>)])
2018-03-09 14:41:52,199 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:52 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:41:53,311 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:53 +0800] "GET /static/image/avatar/c181666905954b57aedff75ddc8d57b6.jpg HTTP/1.1" 200 78361 "-" "Dalvik/2.1.0 (Linux; U; Android 6.0; CAM-AL00 Build/HONORCAM-AL00)"
2018-03-09 14:41:55,473 company_module.py[line:160] INFO 用户598d6ac2de713e32dfc74796没有company_id信息
2018-03-09 14:41:55,500 base.py[line:83] INFO GET http://api.safego.org:9200/sf/daily_report/_search?size=100 [status:200 request:0.026s]
2018-03-09 14:41:55,501 security_module.py[line:1224] INFO elasticsearch query success! at 2018-03-09 14:41:55, user_id:598d6ac2de713e32dfc74796,date:None共计查询到6条记录
2018-03-09 14:41:55,502 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:41:55 +0800] "POST /api/get_daily_info HTTP/1.1" 200 124 "-" "okhttp/3.9.1"
2018-03-09 14:42:50,722 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:42:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:43:50,691 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:43:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:44:50,732 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:44:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:45:54,190 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:45:54 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:46:51,134 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:46:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:47:50,729 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:47:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:48:50,962 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:48:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:49:52,464 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:49:52 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:50:50,730 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:50:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:51:51,780 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:51:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:52:51,770 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:52:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:53:50,714 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:53:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:54:50,720 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:54:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:55:50,744 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:55:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:56:53,303 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:56:53 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:57:50,741 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:57:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:58:50,724 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:58:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 14:59:51,722 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:14:59:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:00:50,734 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:00:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:01:55,410 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:01:55 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:01:55,559 data_view.py[line:244] INFO 2018-03-09 15:01:55.559586: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_15_01_53.zip' ('multipart/form-data')>)])
2018-03-09 15:01:55,562 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:01:55 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:02:50,774 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:02:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:03:51,712 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:03:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:04:50,834 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:04:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:05:51,849 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:05:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:06:50,754 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:06:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:07:50,801 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:07:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:08:50,734 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:08:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:09:51,755 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:09:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:10:50,811 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:10:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:11:50,821 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:11:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:12:50,822 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:12:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:13:50,799 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:13:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:14:51,780 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:14:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:15:51,883 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:15:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:16:51,626 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:16:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:17:50,743 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:17:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:18:50,808 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:18:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:19:50,752 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:19:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:20:51,554 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:20:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:21:50,777 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:21:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:22:53,190 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:22:53 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:22:53,214 data_view.py[line:244] INFO 2018-03-09 15:22:53.213951: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_15_22_51.zip' ('multipart/form-data')>)])
2018-03-09 15:22:53,216 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:22:53 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:23:50,779 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:23:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:24:54,476 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:24:54 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:25:50,757 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:25:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:26:51,091 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:26:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:27:51,759 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:27:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:28:52,718 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:28:52 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:29:52,711 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:29:52 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:30:51,879 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:30:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:31:51,750 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:31:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:32:53,706 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:32:53 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:33:51,413 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:33:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:34:50,760 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:34:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:35:53,631 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:35:53 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:36:52,456 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:36:52 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:37:50,781 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:37:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:38:51,821 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:38:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:39:32,450 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:15:39:32 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 5.1.1; vivo X7 Build/LMY47V; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/57.0.2987.132 MQQBrowser/6.2 TBS/043909 Mobile Safari/537.36 MicroMessenger/6.6.5.1280(0x26060532) NetType/WIFI Language/zh_CN"
2018-03-09 15:39:51,739 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:39:51 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:41:50,815 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:41:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:42:50,791 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:42:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:43:50,808 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:43:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:43:51,645 data_view.py[line:244] INFO 2018-03-09 15:43:51.645464: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_15_43_50.zip' ('multipart/form-data')>)])
2018-03-09 15:43:51,648 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:43:51 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:44:52,421 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:44:52 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:45:50,789 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:45:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:46:50,822 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:46:50 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:47:10,505 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:47:10 +0800] "GET /api/check_version HTTP/1.1" 200 107 "-" "okhttp/3.9.1"
2018-03-09 15:47:13,789 company_module.py[line:160] INFO 用户598d6ac2de713e32dfc74796没有company_id信息
2018-03-09 15:47:13,825 base.py[line:83] INFO GET http://api.safego.org:9200/sf/daily_report/_search?size=100 [status:200 request:0.035s]
2018-03-09 15:47:13,826 security_module.py[line:1224] INFO elasticsearch query success! at 2018-03-09 15:47:13, user_id:598d6ac2de713e32dfc74796,date:None共计查询到6条记录
2018-03-09 15:47:13,827 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:47:13 +0800] "POST /api/get_daily_info HTTP/1.1" 200 124 "-" "okhttp/3.9.1"
2018-03-09 15:47:16,663 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:47:16 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:47:17,541 data_view.py[line:244] INFO 2018-03-09 15:47:17.541034: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_15_47_16.zip' ('multipart/form-data')>)])
2018-03-09 15:47:17,545 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:47:17 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:48:17,646 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:48:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:49:17,738 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:49:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:50:18,915 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:50:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:51:17,475 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:51:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:52:18,094 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:52:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:53:18,619 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:53:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:54:17,482 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:54:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:55:17,470 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:55:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:56:17,618 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:56:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:57:17,461 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:57:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:58:17,491 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:58:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 15:59:17,744 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:15:59:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:00:17,607 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:00:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:01:17,496 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:01:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:02:19,349 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:02:19 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:03:17,628 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:03:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:04:17,963 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:04:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:05:17,483 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:05:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:06:18,261 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:06:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:07:18,402 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:07:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:07:18,939 data_view.py[line:244] INFO 2018-03-09 16:07:18.939350: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_16_07_18.zip' ('multipart/form-data')>)])
2018-03-09 16:07:18,946 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:07:18 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:08:17,702 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:08:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:09:17,939 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:09:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:10:17,956 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:10:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:11:17,525 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:11:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:12:17,460 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:12:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:13:17,677 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:13:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:14:23,560 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:14:23 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:15:18,678 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:15:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:16:17,818 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:16:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:17:18,511 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:17:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:18:17,529 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:18:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:19:17,707 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:19:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:20:17,825 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:20:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:21:17,719 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:21:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:22:17,535 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:22:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:23:22,605 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:23:22 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:24:18,850 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:24:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:25:18,693 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:25:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:26:17,584 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:26:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:27:19,250 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:27:19 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:28:18,313 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:28:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:28:18,943 data_view.py[line:244] INFO 2018-03-09 16:28:18.943507: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_16_28_18.zip' ('multipart/form-data')>)])
2018-03-09 16:28:18,947 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:28:18 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:29:17,641 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:29:17 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:30:18,528 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:30:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:31:19,095 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:31:19 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:32:18,580 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:32:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:33:25,509 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:33:25 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:34:18,667 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:not master
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: not master
2018-03-09 16:34:18,675 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:not master
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: not master
2018-03-09 16:34:18,677 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:34:18 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:35:22,537 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:35:27,545 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:35:27,547 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:35:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:36:22,782 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:36:27,789 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:36:27,791 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:36:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:37:22,875 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:37:27,885 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:37:27,887 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:37:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:38:22,726 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:38:27,737 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:38:27,739 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:38:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:39:23,385 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:39:28,397 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:39:28,398 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:39:28 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:40:23,969 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:40:28,981 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:40:28,983 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:40:28 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:41:22,774 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:41:27,785 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:41:27,787 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:41:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:42:22,586 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:42:27,597 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:42:27,599 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:42:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:43:23,303 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:43:28,315 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:43:28,316 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:43:28 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:44:22,579 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:44:27,589 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:44:27,591 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:44:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:45:22,552 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:45:27,563 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:45:27,565 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:45:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:46:22,788 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:46:27,799 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:46:27,801 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:46:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:47:22,927 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:47:27,937 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:47:27,939 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:47:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:48:22,713 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:48:27,721 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:48:27,723 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:48:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:49:23,583 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:49:28,592 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:49:28,593 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:49:28 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:49:28,594 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:49:28 +0800] "POST /api/add_driving_data HTTP/1.1" 408 92 "-" "okhttp/3.9.1"
2018-03-09 16:49:28,594 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:49:28 +0800] "POST /api/add_driving_data HTTP/1.1" 408 92 "-" "okhttp/3.9.1"
2018-03-09 16:49:28,925 data_view.py[line:244] INFO 2018-03-09 16:49:28.925490: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_16_49_18.zip' ('multipart/form-data')>)])
2018-03-09 16:49:28,930 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:49:28 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:50:23,097 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:50:28,105 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:50:28,107 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:50:28 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:51:24,373 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:51:29,381 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:51:29,382 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:51:29 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:52:23,264 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:52:28,273 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:52:28,275 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:52:28 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:53:22,650 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:53:27,663 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:53:27,665 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:53:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:54:22,962 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:54:27,970 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:54:27,972 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:54:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:55:22,723 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:55:27,731 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:55:27,733 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:55:27 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 16:57:54,857 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:57:59,866 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 16:57:59,867 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:16:57:59 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:01:20,027 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:01:25,033 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:01:25,035 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:01:25 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:04:24,215 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:04:29,225 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:04:29,227 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:04:29 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:08:04,322 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:08:09,333 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:08:09,335 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:08:09 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:10:44,389 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:10:49,399 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:10:49,401 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:10:49 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:10:49,404 data_view.py[line:244] INFO 2018-03-09 17:10:49.404901: user_id: 598d6ac2de713e32dfc74796, files: ImmutableMultiDict([('driving_data', <FileStorage: '2018_03_09_17_10_39.zip' ('multipart/form-data')>)])
2018-03-09 17:10:49,409 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:10:49 +0800] "POST /api/add_driving_data HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:14:03,306 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:14:08,317 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:14:08,319 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:14:08 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:17:03,525 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:17:08,532 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:17:08,534 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:17:08 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:20:15,215 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:20:20,230 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:20:20,232 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:20:20 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 17:23:50,359 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:23:55,366 mongo_db.py[line:913] ERROR mongo_db.save func Error,原因:could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
Traceback (most recent call last):
  File "/home/web/my_env/projects/sq_platform/mongo_db.py", line 904, in save
    save_id = ses.save(save_dict)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 2481, in save
    collation=collation)
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/collection.py", line 767, in _update
    _check_write_command_response([(0, result)])
  File "/home/web/my_env/lib/python3.5/site-packages/pymongo/helpers.py", line 315, in _check_write_command_response
    raise WriteError(error.get("errmsg"), error.get("code"), error)
pymongo.errors.WriteError: could not find host matching read preference { mode: "primary", tags: [ {} ] } for set shard1
2018-03-09 17:23:55,367 mlogging.py[line:130] INFO "210.22.99.54 - - [09/Mar/2018:17:23:55 +0800] "POST /api/gps_push HTTP/1.1" 200 22 "-" "okhttp/3.9.1"
2018-03-09 20:06:26,435 mlogging.py[line:130] INFO "195.154.61.206 - - [09/Mar/2018:20:06:26 +0800] "GET / HTTP/1.1" 302 223 "-" "Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:58.0) Gecko/20100101 Firefox/58.0"
2018-03-09 21:21:51,216 mlogging.py[line:130] INFO "116.62.230.22 - - [09/Mar/2018:21:21:51 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (Linux; Android 6.0.1; vivo Y53 Build/MMB29M) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/38.0.2125.102 Mobile Safari/537.36 VivoBrowser/5.3.3"
2018-03-10 01:01:25,970 mlogging.py[line:130] INFO "119.23.174.205 - - [10/Mar/2018:01:01:25 +0800] "- - HTTP/1.0" 400 84 "-" "-"
2018-03-10 01:06:54,237 mlogging.py[line:130] INFO "47.95.28.143 - - [10/Mar/2018:01:06:54 +0800] "- - HTTP/1.0" 400 84 "-" "-"
2018-03-10 04:01:57,794 mlogging.py[line:130] INFO "47.52.210.90 - - [10/Mar/2018:04:01:57 +0800] "GET / HTTP/1.0" 302 223 "-" "-"
2018-03-10 05:09:24,048 mlogging.py[line:130] INFO "47.97.10.19 - - [10/Mar/2018:05:09:24 +0800] "GET / HTTP/1.0" 302 223 "-" "-"
2018-03-10 06:32:25,133 mlogging.py[line:130] INFO "123.59.54.62 - - [10/Mar/2018:06:32:25 +0800] "- - HTTP/1.0" 400 84 "-" "-"
2018-03-10 06:32:28,269 mlogging.py[line:130] INFO "123.59.54.62 - - [10/Mar/2018:06:32:28 +0800] "- - HTTP/1.0" 0 0 "-" "-"
2018-03-10 06:32:28,354 mlogging.py[line:130] INFO "123.59.54.62 - - [10/Mar/2018:06:32:28 +0800] "GET / HTTP/1.1" 302 223 "-" "Mozilla/5.0 (Windows NT 6.1; WOW64; rv:52.0) Gecko/20100101 Firefox/52.0"
2018-03-10 09:02:23,607 mlogging.py[line:130] INFO "116.62.230.22 - - [10/Mar/2018:09:02:23 +0800] "GET / HTTP/1.0" 302 223 "-" "Mozilla/5.0 (iPad; CPU OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1"
2018-03-10 09:02:23,626 mlogging.py[line:130] INFO "116.62.230.22 - - [10/Mar/2018:09:02:23 +0800] "GET /manage/ HTTP/1.0" 302 233 "-" "Mozilla/5.0 (iPad; CPU OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1"
2018-03-10 09:02:23,648 mlogging.py[line:130] INFO "116.62.230.22 - - [10/Mar/2018:09:02:23 +0800] "GET /manage/index HTTP/1.0" 302 233 "-" "Mozilla/5.0 (iPad; CPU OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1"
2018-03-10 09:02:23,669 mlogging.py[line:130] INFO "116.62.230.22 - - [10/Mar/2018:09:02:23 +0800] "GET /manage/login HTTP/1.0" 200 7906 "-" "Mozilla/5.0 (iPad; CPU OS 9_1 like Mac OS X) AppleWebKit/601.1.46 (KHTML, like Gecko) Version/9.0 Mobile/13B143 Safari/601.1"
